### PR TITLE
My Product Card by GTXtab

### PR DESCRIPTION
**Pull request by GTXtab**
[[DEMO LINK]](https://GTXtab.github.io/layout_product-cards/)
[[TEST REPORT LINK]](https://GTXtab.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,37 @@
...
+      <div class="card" data-qa="card">
+        <img
+          alt = "APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)"
+          class = "card__image"
+          src="images/imac.jpeg"
+        >
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__ratings">
+          <ul class="card__stars-4">
```
Comment (anastasiiavorobiova) [inline src/index.html:21]: Consider using classes from the previous task, the modifier change should work for this block

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,37 @@
...
+            <li class="card__star"></li>
+            <li class="card__star"></li>
+            <li class="card__star"></li>
+            <li class="card__star"></li>
+          </ul>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <div class="card__label">Price:</div>
+          <h1 class="card__value">$2,199</h1>
```
Comment (anastasiiavorobiova) [inline src/index.html:32]:
```suggestion
          <p class="card__value">$2,199</p>
```
The h1 tag is a semantic tag for headings and should be single on the page. The price is not heading and the card element wouldn't be the single card.

File: src/index.html [hunk 3]
```diff
@@ -8,6 +8,37 @@
...
+        </div>
+        <a
+          class="card__button"
+          data-qa="hover"
+          href="#"
+        >
+        Buy
```
Comment (anastasiiavorobiova) [inline src/index.html:39]:
```suggestion
          Buy
```
Consider fixing indents according to the code style guide

File: src/index.html [hunk 4]
```diff
@@ -8,6 +8,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="main">
+      <div class="card" data-qa="card">
+        <img
+          alt = "APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)"
```
Comment (anastasiiavorobiova) [inline src/index.html:14]:
```suggestion
          alt="APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)"
```

File: src/index.html [hunk 5]
```diff
@@ -8,6 +8,37 @@
...
+          class = "card__image"
```
Comment (anastasiiavorobiova) [inline src/index.html:15]:
```suggestion
          class="card__image"
```

File: src/style.scss [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
```
Comment (anastasiiavorobiova) [inline src/style.scss:21]:
```suggestion

```
Avoid limiting the height of the card, it should depend on the content in this case

File: src/style.scss [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+    width: $--standard-width;
+    display: flex;
+    align-self: center;
+    align-items: center;
+    justify-content: space-between;
+    height: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__stars-4 {
```
Comment (anastasiiavorobiova) [inline src/style.scss:63]: Consider using full styling from the previous task

File: src/style.scss [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+    margin: 0;
+  }
+
+  &__price {
+    width: $--standard-width;
+    display: flex;
+    align-items: center;
+    align-self: center;
+    justify-content: space-between;
+    height: 18px;
```
Comment (anastasiiavorobiova) [inline src/style.scss:97]:
```suggestion

```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added comments to fix

File: src/style.scss (hunk 7 above)
Comment (GTXtab) [reply src/style.scss:63]: is it neccesary to take "copied-code" from the previous? Because i cant connect it with my already made code.

**Comment by GTXtab** [comment]
I have done all things which you want. I will be thankful if u accept
these 2 repo :)
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3615 ,
https://github.com/mate-academy/layout_product-cards/pull/3608

вс, 10 сент. 2023 г. в 16:32, anastasiiavorobiova ***@***.***

**Pull request closed without merging**